Java/.codesandbox/template.json

```diff
@@ -0,0 +1,7 @@
+{
+  "title": "Unleash C# SDK",
```

Comment:
Not java?

Reply:
Haven't started the setup for Java yet; just copied code from dotnet.